nibabel/filebasedimages.py

```diff
@@ -511,3 +512,79 @@ def path_maybe_image(klass, filename, sniff=None, sniff_max=1024):
         if sniff is None or len(sniff[0]) < klass._meta_sniff_len:
             return False, sniff
         return klass.header_class.may_contain_header(sniff[0]), sniff
+
+
+class SerializableImage(FileBasedImage):
+    '''
+    Abstract image class for (de)serializing images to/from byte strings.
+
+    The class doesn't define any image properties.
+
+    It has:
+
+    methods:
+
+       * .to_bytes() - serialize image to byte string
+
+    classmethods:
+
+       * from_bytes(bytestring) - make instance by deserializing a byte string
+
+    The following properties should hold:
```

Comment:
How do you want this to look in the rendered doc?  What happens with these indented lists?

Reply:
I actually haven't built the docs here. I'll fiddle with this.